File: Chuong 11 - Ke Thua/Ung Dung Quan Ly Thu Vien C++ - C# - Java/Source Code/Java/Java_QuanLyDocGia/src/java_quanlydocgia/CNgay.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package java_quanlydocgia;

import java.util.Scanner;

/**
 *
 * @author NguyenVietNamSon
 */
public class CNgay {
    private int Ngay, Thang, Nam;

        public void Nhap()
        {
            Scanner scan = new Scanner(System.in);
            System.out.print("\nNhap ngay: ");
            Ngay = Integer.parseInt(scan.nextLine());

            System.out.print("\nNhap thang: ");
            Thang = Integer.parseInt(scan.nextLine());

            System.out.print("\nNhap nam: ");
            Nam = Integer.parseInt(scan.nextLine());
        }

        public void Xuat()
        {
            System.out.println("Ngay " + String.valueOf(Ngay) + " thang " + String.valueOf(Thang) + " nam " + String.valueOf(Nam));
        }
}

File: Chuong 11 - Ke Thua/Ung Dung Quan Ly Thu Vien C++ - C# - Java/Source Code/C#/Quan Ly Doc Gia/QuanLyDocGia/CNgay.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyDocGia
{
    class CNgay
    {
        private int Ngay, Thang, Nam;

        public void Nhap()
        {
            Console.Write("\nNhap ngay: ");
            Ngay = int.Parse(Console.ReadLine());

            Console.Write("\nNhap thang: ");
            Thang = int.Parse(Console.ReadLine());

            Console.Write("\nNhap nam: ");
            Nam = int.Parse(Console.ReadLine());
        }

        public void Xuat()
        {
            Console.Write("Ngay {0} thang {1} nam {2}", Ngay, Thang, Nam);
        }
    }
}
